pkg/storage/tsdb/config.go

```diff
@@ -34,14 +35,34 @@ type Config struct {
 	GCS gcs.Config `yaml:"gcs"`
 }
 
+// RangeList is the block ranges for a tsdb
+type RangeList []time.Duration
```

Comment:
What do you think about naming the table `DurationList` since it's just a list of `time.Duration`?

Reply:
Fair enough